### ISSUE TITLE
Support for detectron2

### ISSUE THREAD

**Issue by petoor** [grimoire/mmdetection-to-tensorrt#57]
Hi.

Are there any plans to extend this repo to also transform detectron2 models to tensorrt?

**Comment by grimoire** [comment]
No, That's an attractive idea, But I'm afraid I do not have enough time to finish it.
I plan to add a tutorial about how to add convertor or module support. That might help those who want to create tools for detectron2.
